invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/CustomFieldSerializer.js

```diff
@@ -0,0 +1,69 @@
+// This file is part of React-Invenio-Deposit
+// Copyright (C) 2020-2022 CERN.
+// Copyright (C) 2020 Northwestern University.
+//
+// React-Invenio-Deposit is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import _get from "lodash/get";
+import _set from "lodash/set";
+import _cloneDeep from "lodash/cloneDeep";
+import _isArray from "lodash/isArray";
+
+export class CustomFieldSerializer {
```

Comment:
This is copied from react-invenio-deposit. Shall we move it to react-invenio-forms?